apps/server/src/modules/task-card/controller/dto/task-card.params.ts

```diff
@@ -63,15 +64,15 @@ export class CardElementParams {
 export class TaskCardParams {
 	@IsString()
 	@IsMongoId()
-	@IsOptional()
-	@ApiPropertyOptional({
+	@ApiProperty({
 		description: 'The id of an course object.',
 		pattern: '[a-f0-9]{24}',
 	})
-	courseId?: string;
+	courseId!: string;
 
 	@IsString()
 	@MinLength(2)
```

Comment:
ticket says `min length 1 Zeichen`
which is not reflected here

Reply:
I think this validation will never reached because of "IsMongoId". But to be clear I can add this.

Reply:
right